test/extended/networking/util.go

```diff
@@ -417,3 +419,120 @@ func InOVNKubernetesContext(body func()) {
 		},
 	)
 }
+
+// SubnetIPs enumerates all subnets in an ip net (starting with the provided IP address).
+func SubnetIPs(ipnet net.IPNet) ([]net.IP, error) {
```

Comment:
code, to me, seems fine. first thing I wondered was if these functions aren't already available somewhere/somehow
in some other utility or library. but, assuming you already checked and nothing obvious stood out.

Reply:
I already checked and couldn't find anything (which doesn't mean that it's not there). I'll remove code duplication for DetermineImageFromRelease, though.